kolibri/plugins/default_theme/kolibri_plugin.py

```diff
@@ -25,4 +25,25 @@ def theme(self):
                     "style": "padding-left: 64px; padding-right: 64px; margin-bottom: 8px; margin-top: 8px",
                 },
             },
+            "logos": [
+                {
+                    "src": static("kolibri-logo.svg"),
+                    "content_type": "image/svg+xml",
+                    # See https://web.dev/maskable-icon/ for details on what
+                    # icons count as maskable. The default Kolibri logo is not,
+                    # as the outer ‘waves’ circle gets cropped.
```

Comment:
I think these forward ticks are causing the non-ASCII character error in the Python 2.7 tests.